### PR TITLE
Update installation notes

### LINKED ISSUE #61: Docs are potentially confusing for installation
### Desired capability or behavior.

Both the readme and docs could be more clear about the installation process.
Tests are not explicitly required to run Aviary.
Make this clear in the docs.

### Is your feature request related to a problem? Please describe.

_No response_

### Associated Bug Report

_No response_

### PR DESCRIPTION
**Pull request by johnjasa**
### Summary

Improves installation docs.

### Related Issues

- Resolves #61

### Backwards incompatibilities

None

### New Dependencies

None

### CODE REVIEW

**Review by jdgratz10 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -67,4 +76,4 @@ We have also provided a static version of the `environment.yml` at the top level
 ## Planned future features
 
 Aviary is in active development.
-We plan to expand its capabilities and add features as detailed in [this doc page](https://openmdao.github.io/om-Aviary/misc_resources/planned_future_features.html).
+We plan to expand its capabilities and add features as detailed in [this doc page](https://openmdao.github.io/om-Aviary/misc_resources/planned_future_features.html).
```
Comment (crecine) [inline README.md:79]: generally I prefer more descriptive hyperlink text than "here" or "this page", maybe something like: 
```suggestion
We plan to expand its capabilities and have provided a non-exhaustive [list of future features](https://openmdao.github.io/om-Aviary/misc_resources/planned_future_features.html).
```

File: aviary/docs/getting_started/installation.md [hunk 2]
```diff
@@ -2,16 +2,33 @@
...
+
+If you want to also run the Aviary test suite (this is not strictly necessary), you can instead run:
+
+    pip install .[test]
+
+If you also want to install all packages used for the Aviary tests _and_ external subsystem examples, you can instead run:
+
+    pip install .[all]
+
+If you are a developer and plan to modify parts of the Aviary code, install in an "editable mode" with ``pip``:
```
Comment (crecine) [inline aviary/docs/getting_started/installation.md:18]: Is there a reason pip is ``pip`` here but just pip on line 5?

File: aviary/docs/getting_started/installation.md [hunk 3]
```diff
@@ -198,7 +215,7 @@ Successfully installed pyoptsparse-2.10.1
 
 ### Installing Aviary and Running Tests
 
-Now, we are ready to install Aviary. Assume that you will become a contributor sonner or later, we want to install a copy from the main source. Assume you have a GitHub account. Let us open `https://github.com/openMDAO/om-aviary/` in a web browser and click [fork](https://github.com/OpenMDAO/om-Aviary/fork) on the top-right corner. You then have created your own copy of Aviary on GitHub website. Now we create a copy on your local drive (supposed `USER_ID` is your GitHub account ID):
+Now, we are ready to install Aviary. Assume that you will become a contributor sooner or later, we want to install a copy from the main source. Assume you have a GitHub account. Let us open `https://github.com/openMDAO/om-aviary/` in a web browser and click [fork](https://github.com/OpenMDAO/om-Aviary/fork) on the top-right corner. You then have created your own copy of Aviary on GitHub website. Now we create a copy on your local drive (supposed `USER_ID` is your GitHub account ID):
```
Comment (crecine) [inline aviary/docs/getting_started/installation.md:218]:
```suggestion
Now, we are ready to install Aviary. Assuming that you will become a contributor sooner or later, we want to install a copy from the main source. (You will need a GitHub account for this) Let us open `https://github.com/openMDAO/om-aviary/` in a web browser and click [fork](https://github.com/OpenMDAO/om-Aviary/fork) on the top-right corner. You then have created your own copy of Aviary on GitHub website. Now we create a copy on your local drive (supposing `USER_ID` is your GitHub account ID):
```

**Review by crecine — APPROVED** [review]

File: aviary/docs/getting_started/installation.md (hunk 2 above)
Comment (johnjasa) [reply aviary/docs/getting_started/installation.md]: Nope, changed!

**Pull request merged**